bindings/wasm/examples/worker.ts

```diff
@@ -457,8 +460,29 @@ async function exportGLB(manifold?: Manifold) {
     wrapper.addChild(node);
   }
 
+  const results: {[name: string]: string|null} = {};
+  if (manifold != null) {
+    const mesh = manifold.getMesh();
+    let vertices = new Float32Array(mesh.numVert * 3);
+    for (let i = 0; i < mesh.numVert; ++i) {
+      for (let j = 0; j < 3; ++j)
+        vertices[i * 3 + j] = mesh.vertProperties[i * mesh.numProp + j];
+    }
+    const model =
+        to3dmodel({simple: [{vertices, indices: mesh.triVerts, id: '1'}]});
```

Comment:
I think we can, but I am not very familiar with this. We probably need to look into the 3mf spec about how to specify the hierarchy and materials.

Reply:
I think this may get complicated. What about leaving this to a follow up PR?

Reply:
Yeah, actually it looks like we also need a small change in `to3dmodel` - @hrgdavor where can we file issues and/or make PRs to `3mf-export`? Looking at your code, it appears a component tree can't be exported correctly because it places every component in the build, rather than just the root.

Reply:
code is in this repo. Suggestions and PR's are welcome https://github.com/hrgdavor/jscadui/
3mf export part is here https://github.com/hrgdavor/jscadui/tree/main/file-format/3mf-export